ui/pages/swaps/prepare-swap-page/smart-transactions-popover.tsx

```diff
@@ -22,6 +20,8 @@ import {
   ButtonLink,
   ButtonLinkSize,
 } from '../../../components/component-library';
+import { ModalContent } from '../../../components/component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../components/component-library/modal-header/deprecated';
 import { SMART_SWAPS_FAQ_AND_RISK_DISCLOSURES_URL } from '../../../../shared/constants/swaps';
 
 interface Props {
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 5 10 03 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/900f7582-dda3-4f41-a800-9910f2aedc9e">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 5 10 13 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c44bd10c-9b34-4780-ab93-945bb4ea6bf4">